olap-examples/gcp-setup/OLAP_GCP.md

```diff
@@ -0,0 +1,55 @@
+# GCP Spark Cluster Setup
+
+To create a spark cluster in GCP, the following can be used from command line.
+
+```bash
+dataproc_name="example-spark-cluster-gcp"
+region=us-central1
+zone=us-central1-a
+instance_type=n2d-highmem-8
```

Comment:
`n2d-highmem-8` has twice the memory (64GB) compared to AWS `m7a.2xlarge` is that on purpose?

Reply:
Good call out, this was not on purpose i will fix sometime shortly.